bank_integration/bank_integration/api/bank_api.py

```
@@ -190,6 +219,41 @@ def delete_cache(self):
+            time.sleep(0.5)
+        return None, None
+
+    def cleanup_download_dir(self):
+        """Remove the temp download directory and its contents."""
+        import shutil
+
+        if hasattr(self, "download_dir") and self.download_dir:
+            try:
+                shutil.rmtree(self.download_dir, ignore_errors=True)
+            except Exception:
+                pass
```

Ruff (0.15.5)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)